python/lsst/meas/extensions/shapeHSM/_hsm_shape.py

```diff
@@ -67,6 +54,37 @@ class HsmShapeConfig(measBase.SingleFramePluginConfig):
         default=["BAD", "SAT"],
     )
 
+    @property
+    def shearType(self):
+        """Base class property for the desired method of PSF correction.
+
+        The following options are available through GalSim. The first three
+        options return an e-type distortion, whereas the last option returns a
+        g-type shear:
+
+        - "REGAUSS": Regaussianization method from Hirata & Seljak (2003).
+        - "LINEAR": A modification by Hirata & Seljak (2003) of methods in
+          Bernstein & Jarvis (2002).
+        - "BJ": The method developed by Bernstein & Jarvis (2002).
+        - "KSB": The method from Kaiser, Squires, & Broadhurst (1995).
+
+        The default method is "REGAUSS". Subclasses can override this property,
+        but it cannot be set externally, making it effectively read-only. The
+        setter is deprecated and will not do anything other than emit a
+        deprecation warning.
+        """
+        return "REGAUSS"
```

Comment:
Raise `NotImplementedError` here saying that this must be implemented in the subclasses.